lib/src/Base/Common/ResourceMap.cxx

```diff
@@ -1380,6 +1380,9 @@ void ResourceMap::loadDefaultConfiguration()
   addAsUnsignedInteger("AdaptiveDirectionalStratification-DefaultMaximumStratificationDimension", 3);
   addAsUnsignedInteger("AdaptiveDirectionalStratification-DefaultNumberOfSteps", 2);
 
+  // Analytical parameters //
+  addAsScalar("Analytical-DefaultLimitStateTolerance", 0.01);
```

Comment:
I do not know a perfect method to set the default tolerance, but this seems quite loose to me. Would it be relevant to be more stringent, e.g. $10^{-5}$?

Reply:
This value is very dependent of the optimization problem formulation and physics involved. 
I agree with you about the need to name it "Analytical-DefaultLimitStateAbsoluteTolerance".